### DISCUSSION TITLE
Wrong mouse cursor on the note pane scrollbar

### DISCUSSION THREAD

**Discussion by virtuallyvlad** [standardnotes/forum#2475] [General]
On both the desktop app and the web app, the mouse cursor when using the text scrollbar is wrong while the **Plain Editor** is used: it has the "text select" style cursor instead of the default/pointer cursor.

Meanwhile, the cursors on the other scrollbars, the left/tag pane and the middle/list pane, correctly use the default/pointer cursor.

**Comment by JaspalSuri** [comment]
Hi @virtuallyvlad, thank you for bringing this issue to our attention. I can reproduce it in the web app but not in the desktop app on Windows 10 in the latest beta (v3.9.0b1) can you export a set of backups of your notes via the Account menu, [install the beta,](https://github.com/standardnotes/desktop/releases/tag/v3.9.0-beta01) and verify whether the issue persists?

**Comment by virtuallyvlad** [comment]
Yup, it still does it in v3.9.0b1.  Tested that beta on win7 and win10; both tests had the wrong cursor over that scrollbar.

FYI this is with the **Plain Editor**.  Other editors may not have this issue.

Also, perhaps this should be moved to https://github.com/standardnotes/forum/issues as it applies to both desktop & web.

**Comment by JaspalSuri** [comment]
>FYI this is with the Plain Editor.

Thank you for sharing that! I was able to reproduce the issue. 

>Yup, it still does it in v3.9.0b1. Tested that beta on win7 and win10; both tests had the wrong cursor over that scrollbar.

Thank you for testing the beta app as well. Please update to the latest stable (in order to receive future updates automatically) from [here](https://standardnotes.org/download). 

>Also, perhaps this should be moved to [the forum](https://github.com/standardnotes/forum/issues) as it applies to both desktop & web.

In this case, if it affects both apps, then it belongs in the web app's repo as the web app precedes the desktop app in development. 🙂

**Comment by virtuallyvlad** [comment]
Haha, I can't believe this is still around and I can't believe no one else notices!  ミ●﹏☉ミ

![image](https://user-images.githubusercontent.com/37453320/202321042-86d3368d-d539-48d1-8a6b-132a1d2b3753.png)

**Comment by myreli** [comment]
As of version `3.148.0` it seems to be resolved. Please let us know if you find any other issues.

https://user-images.githubusercontent.com/17554234/218527425-5c6545c6-653a-424f-8cea-df1b4950f976.mp4

**Reply by virtuallyvlad** [reply]
> As of version `3.148.0` it seems to be resolved. Please let us know if you find any other issues.

Nope, as of v3.149.5 (desktop app) on win7 it's still an issue.  Is that macOS?  If so, maybe it's just an issue on windows.
